apps/common-app/src/examples/ScrollToExample.tsx

```diff
@@ -24,8 +39,35 @@ export default function ScrollToExample() {
     })();
   };
 
+  const examples = [
+    {
+      title: 'ScrollView',
+      component: ScrollViewExample,
+    },
+    {
+      title: 'FlatList',
+      component: FlatListExample,
+    },
+    {
+      title: 'FlashList',
+      component: FlashListExample,
+    },
+  ];
+
+  const Example = examples[currentExample].component;
+
   return (
     <>
+      <View style={styles.optionsRow}>
+        {examples.map(({ title }, index) => (
+          <Button
+            disabled={index === currentExample}
+            key={title}
+            title={title}
+            onPress={() => setCurrentExample(index)}
+          />
+        ))}
+      </View>
       <View style={styles.buttons}>
         <Switch
```

Comment:
I think it would be great to let example user know what the switch does 😄 Some small text will be sufficient

Reply:
Makes sense to change this description. It was also confusing to me when I started modifying this example.